docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -135,12 +136,17 @@ Emitted when the ownership is transferred.
 :RoleRevoked: xref:#IAccessControl-RoleRevoked[RoleRevoked]
 :RoleAdminChanged: xref:#IAccessControl-RoleAdminChanged[RoleAdminChanged]
 
-[.hljs-theme-light.nopadding]
 ```javascript
 use openzeppelin::access::accesscontrol::interface::IAccessControl;
 ```
 
-External interface of AccessControl declared to support https://github.com/starknet-io/SNIPs/blob/main/SNIPS/snip-5.md[SRC5] detection.
+External interface of AccessControl.
+
+[.contract-index]
+.Interface ID
+--
+0x23700be02858dbe2ac4dc9c9f66d0b6b0ed81ec7f970ca6844500a56ff61751
+--
```

Comment:
should we add a link to SRC5's section for anyone who appears in here without enough context? it should be easy to maintain if we're already standardizing this "interface id" section across APIs.

Reply:
I would add them on the introspection PR, because the section doesn't exist on this one. I added a link just to confirm if you like the approach, let me know (the link is broken here, but should work after merging with the other PR).